modules/account-lib/test/unit/coin/dot/transactionBuilder/transactionBuilderFactory.ts

```diff
@@ -88,10 +88,10 @@ describe('dot Transaction Builder Factory', () => {
     const builder = factory.from(rawTx.proxy.signed);
     should(builder).instanceOf(TransferBuilder);
     builder
-      .validity({ firstValid: 3933 })
+      .validity({ firstValid: 3933, maxDuration: 64 })
```

Comment:
What's the reason behind adding the `maxDuration` param here? It is because of the updates in #1867?

Reply:
I generated the new proxy txHex using this test here https://github.com/BitGo/BitGoJS/blob/77d800ca23e845fdc481e35790524c958c9f52a9/modules/account-lib/test/unit/coin/dot/transactionBuilder/transferBuilder.ts#L223

This test was failing since the txHex wasnt matching and it was because I had to add `maxDuration: 64`  since the other transaction was being built like this.